pkg/testing/fixture_install.go

```diff
@@ -221,15 +231,17 @@ func (f *Fixture) collectDiagnostics() {
 		f.t.Logf("failed to collect diagnostics; failed to create %s: %s", diagPath, err)
 		return
 	}
-	outputPath := filepath.Join(diagPath, fmt.Sprintf("%s-diagnostics-%s.zip", f.t.Name(), time.Now().Format(time.RFC3339)))
+	outputPath := filepath.Join(diagPath,
+		fmt.Sprintf("%s-diagnostics-%s.zip", f.t.Name(), time.Now().Format(time.RFC3339)))
 
 	output, err := f.Exec(ctx, []string{"diagnostics", "-f", outputPath})
 	if err != nil {
-		f.t.Logf("failed to collect diagnostics to %s (%s): %s", outputPath, err, output)
+		f.t.Logf("failed to collect diagnostics to %s. err: %v: output: %q",
+			outputPath, err, output)
 	}
 }
 
-func collectDiag() bool {
+func collectDiagFlag() bool {
```

Comment:
If you're renaming this function, you might as well rename `keepInstalled` -> `keepInstalledFlag` for consistency.